tests/test_recipe_synth_basics.py

```
@@ -453,3 +454,201 @@ def test_synthesizer_default_templates_structure(self):
+        # All values should be positive
+        for value in nutrition.values():
+            assert isinstance(value, (int, float))
+            assert value >= 0
+
+    def test_adjust_for_servings(self, synthesizer):
+        """Test _adjust_for_servings method."""
+        ingredients = [
+            {"name": "tomatoes", "amount": 200, "unit": "g"},
+            {"name": "pasta", "amount": 100, "unit": "g"},
+        ]
+        original_servings = 2
```

Ruff (0.14.1)

Local variable `original_servings` is assigned to but never used

Remove assignment to unused variable `original_servings`

(F841)